examples/ts-angular-chat/src/styles.css

```
@@ -0,0 +1,510 @@
+  --surface: rgba(255, 255, 255, 0.045);
+  --surface-strong: rgba(255, 255, 255, 0.07);
+  --border: rgba(255, 255, 255, 0.09);
+  --border-strong: rgba(255, 255, 255, 0.16);
+  --accent: #ff7a18;
+  --accent-2: #ffb347;
+  --accent-soft: rgba(255, 122, 24, 0.16);
+  --text: #f6f1ea;
+  --muted: #9b9088;
+  --muted-dim: #6f655d;
+
+  --font-display: 'Instrument Serif', Georgia, serif;
```

Stylelint (17.13.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)